platform/core/src/utils/Combine.ts

```diff
@@ -0,0 +1,36 @@
+const handler = {
+  get: (target, prop, receive) => {
+    for (const obj of target) {
+      if (prop in obj) {
+        return obj[prop];
+      }
+    }
+  },
+
+  set: (target, prop, value) => {
+    target[0][prop] = value;
```

Comment:
Due to the way that we set the first object when we use `set()` method, I think we need to add an explicit warning about the possible changes to the original objects used to create the proxy when use `set()` method on the proxy.

Or instead, we can have internal "empty" target object that sits at the index:0, removing the need of using empty object(`{}`) when we call `Combine()`

```
Combine({}, obj1, obj2, ...)
```

Reply:
Added.